app-tasks/rf/src/rf/uploads/modis/create_geotiff.py

```diff
@@ -37,20 +66,13 @@ def create_geotiffs(modis_path, output_directory):
                      '-a_nodata', '-28672',
```

Comment:
I don't think this is used/created, which causes an error down below for the overview generation.